src/semidiscretization/semidiscretization_hyperbolic_parabolic.jl

```diff
@@ -3,54 +3,81 @@
+
+    function SemidiscretizationHyperbolicParabolic{Mesh, Equations, EquationsParabolic,
+                                                   InitialCondition, BoundaryConditions,
+                                                   BoundaryConditionsParabolic,
+                                                   SourceTerms, Solver, SolverParabolic,
+                                                   Cache, CacheParabolic}(mesh::Mesh,
+                                                                          equations::Equations,
+                                                                          equations_parabolic::EquationsParabolic,
+                                                                          initial_condition::InitialCondition,
+                                                                          boundary_conditions::BoundaryConditions,
+                                                                          boundary_conditions_parabolic::BoundaryConditionsParabolic,
+                                                                          source_terms::SourceTerms,
+                                                                          solver::Solver,
+                                                                          solver_parabolic::SolverParabolic,
+                                                                          cache::Cache,
+                                                                          cache_parabolic::CacheParabolic) where {
+                                                                                                                  Mesh,
+                                                                                                                  Equations,
+                                                                                                                  EquationsParabolic,
+                                                                                                                  InitialCondition,
+                                                                                                                  BoundaryConditions,
+                                                                                                                  BoundaryConditionsParabolic,
+                                                                                                                  SourceTerms,
+                                                                                                                  Solver,
+                                                                                                                  SolverParabolic,
+                                                                                                                  Cache,
+                                                                                                                  CacheParabolic
```

Comment:
```suggestion
    function SemidiscretizationHyperbolicParabolic{Mesh, Equations, EquationsParabolic,
                                                   InitialCondition, BoundaryConditions,
                                                   BoundaryConditionsParabolic,
                                                   SourceTerms, Solver, SolverParabolic,
                                                   Cache, CacheParabolic
                                                   }(mesh::Mesh, equations::Equations,
                                                      equations_parabolic::EquationsParabolic,
                                                      initial_condition::InitialCondition,
                                                      boundary_conditions::BoundaryConditions,
                                                      boundary_conditions_parabolic::BoundaryConditionsParabolic,
                                                      source_terms::SourceTerms,
                                                      solver::Solver,
                                                      solver_parabolic::SolverParabolic,
                                                      cache::Cache,
                                                      cache_parabolic::CacheParabolic
                                                      ) where {Mesh, Equations, EquationsParabolic,
                                                               InitialCondition, BoundaryConditions,
                                                               BoundaryConditionsParabolic,
                                                               SourceTerms, Solver,
                                                               SolverParabolic, Cache,
                                                               CacheParabolic}
```
Certainly better than the abomination before, I'd say.

Reply:
What you suggested didn't work, since JF insisted on putting the closing `) where {` on the same line as the last function argument. 34a13fcf5 is still a small improvement IMHO.